### PR TITLE
W-14975729-SFTP-2.4-Document-Identity-file-issue-KS

### PR DESCRIPTION
**Pull request by Kabilan-s-cx**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by dipeshkumardutta-sfemu — DISMISSED** [review]
LGTM

File: sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc [hunk 1]
```diff
@@ -109,6 +109,24 @@ Here is a list of common throws messages and how to interpret them.
 
   The file to write was deleted.
 
+== Troubleshoot SFTP Connector Issues with Identity File in .ppk Format
+
+When using the SFTP Connector version 2.0.0 and later with the Preferred Authentication Method set to `PUBLIC_KEY`, you might face issues if the Identity File is in `.ppk` format. This is due to a limitation with the `Apache Mina library`, which does not support the `.ppk` file format, causing the application to fail.
```
Comment (Crispy-Salesforce) [inline sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc]:
```suggestion
When using the SFTP Connector version 2.0.0 and later with the preferred authentication method set to `PUBLIC_KEY`, you can face issues if the `Identity` file is PPK. The `Apache Mina library` doesn’t support the PPK file format, so the application fails.
```

Comment (Crispy-Salesforce) [reply sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc]: Look at the file types and file extensions, and font choices in our style guide:
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#:~:text=file%20types%20and%20file%20extensions

File: sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc [hunk 2]
```diff
@@ -109,6 +109,24 @@ Here is a list of common throws messages and how to interpret them.
 
   The file to write was deleted.
 
+== Troubleshoot SFTP Connector Issues with Identity File in .ppk Format
```
Comment (Crispy-Salesforce) [inline sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc]:
```suggestion
== Troubleshoot SFTP Connector Issues with a PPK Identity File 
```

File: sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc [hunk 3]
```diff
@@ -109,6 +109,24 @@ Here is a list of common throws messages and how to interpret them.
...
+
+When using the SFTP Connector version 2.0.0 and later with the Preferred Authentication Method set to `PUBLIC_KEY`, you might face issues if the Identity File is in `.ppk` format. This is due to a limitation with the `Apache Mina library`, which does not support the `.ppk` file format, causing the application to fail.
+
+To resolve this issue:
+
+. Convert the Identity File format from `.ppk` to `.pem` using PuTTYgen.
```
Comment (Crispy-Salesforce) [inline sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc]:
```suggestion
. Convert the PPK `Identity` file to PEM by using `PuTTYgen`.
```

File: sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc [hunk 4]
```diff
@@ -109,6 +109,24 @@ Here is a list of common throws messages and how to interpret them.
...
+
+. Update the configuration file to use the `.pem` file.
```
Comment (Crispy-Salesforce) [inline sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc]:
```suggestion
. Update the configuration file to use the PEM file.
```

**Review by Crispy-Salesforce — DISMISSED** [review]

File: sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc (hunk 1 above)
Comment (Kabilan-s-cx) [reply sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc]: Thank you @Crispy-Salesforce. I wasn’t aware of this part.

File: sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc [hunk 5]
```diff
@@ -1,6 +1,6 @@
 = Troubleshooting SFTP Connector - Mule 4
 
-To troubleshoot Anypoint Connector for SFTP (SFTP Connector), become familiar with the information about troubleshooting connectivity, permissions and server-side issues, troubleshooting Mule application issues, and interpreting commonly thrown messages.
+To troubleshoot Anypoint Connector for SFTP (SFTP Connector), become familiar with the information about troubleshooting connectivity, permissions and server-side issues, troubleshooting Mule application issues, interpreting commonly thrown messages, and troubleshooting identity file format issues.
```
Comment (Crispy-Salesforce) [inline sftp/2.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc]:
```suggestion
To troubleshoot Anypoint Connector for SFTP (SFTP Connector), become familiar with the information about troubleshooting connectivity, permissions, and server-side issues, troubleshooting Mule application issues, interpreting commonly thrown messages, and troubleshooting identity file format issues.
```

**Review by Crispy-Salesforce — DISMISSED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**